.github/workflows/build_and_test.yml

```diff
@@ -578,12 +578,12 @@ jobs:
     strategy:
       matrix:
         include:
-          - os: "windows-2019"
-            platform: "Visual Studio 16 2019"
+          - os: "windows-2025"
+            platform: "Visual Studio 17 2022"
             arch: "x64"
             pcap_lib: "npcap"
-          - os: windows-2019
-            platform: "Visual Studio 16 2019"
+          - os: windows-2025
+            platform: "Visual Studio 17 2022"
```

Comment:
I don't think we need to run VS2022 on both versions of Windows. Since VS2019 is not supported in the newer images, we unfortunately need to drop support for VS2019.

We can keep VS2022 running on `windows-2025`